src/main/java/cms/gov/madie/measure/models/TestCase.java

```diff
@@ -10,6 +10,7 @@
 public class TestCase {
   private String id;
   private String name;
```

Comment:
@gregory-akins we are adding a title, does this mean we no longer need `name`?

Reply:
That's by understanding. `name` is now deprecated in favor of `title`.

Reply:
Should this PR address the deprecated name attribute?

Reply:
if it is a minor change, we can address it in this PR otherwise new ticket is fine too. Also, branch seems to be behind develop

Reply:
The field isn't exposed. Removing it could be done in another PR.